app/controllers/questions/mailing_address_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Questions
+  class MailingAddressController < QuestionsController
+    before_action :set_default_mailing_address, only: %i[edit]
+
+    private
+
+    def section_title
+      "Personal Information"
+    end
+
+    def illustration_path; end
+
+    def set_default_mailing_address
```

Comment:
another option may be to set these fields on the intake when we create it (since we'll already have them)

Reply:
I like that idea, since we'll end up with `Intake` records that always have these fields set, which is a simpler assumption.

Reply:
✅